android/demos/rib-workers/src/main/kotlin/com/uber/rib/workers/root/RootActivity.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2023. Uber Technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.uber.rib.workers.root
+
+import android.os.Bundle
+import android.view.ViewGroup
+import com.uber.rib.core.RibActivity
+import com.uber.rib.core.ViewRouter
+import com.uber.rib.core.WorkerBinder
+import motif.Creatable
+import motif.Expose
+import motif.NoDependencies
+import motif.ScopeFactory
+
+class RootActivity : RibActivity() {
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    WorkerBinder.initializeMonitoring { println(it) }
```

Comment:
WorkerBinder monitoring initialization here

Reply:
Good point, will defined a more real case sample of monitoring (e.g. warn on debug builds of long UI durations) and initialized at RootInteractor.